src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -735,6 +722,21 @@ function MoneyRequestConfirmationList({
                 setDidConfirm(true);
                 onConfirm?.(selectedParticipants);
             }
+
+            if (formError) {
+                return;
+            }
+
+            if (iouType === CONST.IOU.TYPE.PAY) {
```

Comment:
This code feels a bit weird to me. Why isn't this just an else?
```suggestion
            else {
                if (formError) {
                    return;
                }
```

Reply:
I'm coming from that this check 
<img width="284" alt="image" src="https://github.com/Expensify/App/assets/87341702/0de0e88d-693e-47f1-9043-15e16d0123ad">

only matters for `if (iouType === CONST.IOU.TYPE.PAY) {` according to the code.